antenna/app.py

```diff
@@ -302,17 +317,22 @@ def get_app(config=None):
     # Set up metrics
     setup_metrics(app_config('metrics_class'), config, logger)
 
-    # Build the app, add resources, and log the rest of things
+    # Build the app and heartbeat manager
     app = AntennaAPI(config)
 
+    # Add restources
     app.add_route('breakpad', '/submit', BreakpadSubmitterResource(config))
     app.add_route('version', '/__version__', VersionResource(config, basedir=app_config('basedir')))
     app.add_route('heartbeat', '/__heartbeat__', HeartbeatResource(config, app))
     app.add_route('lbheartbeat', '/__lbheartbeat__', LBHeartbeatResource(config))
     app.add_route('broken', '/__broken__', BrokenResource(config))
 
+    # Finish logging configuration
     log_config(logger, app)
 
+    # Verify that we're ready to start
+    app.verify()
```

Comment:
Now all verification happens after everything has been created but before Antenna is handling incoming connections.

Exceptions that are kicked up here will prevent Antenna from starting and will get sent to Sentry since Sentry is configured to listen to logged exceptions.